docs/book/src/reference/envtest.md

```diff
@@ -151,6 +151,16 @@ testEnv = &envtest.Environment{
 }
 ```
 
+## Installing the cert-manager
+
+To test the webhooks, `cert-manager` needs to be installed in the test server. This will help for provisioning the certificates for the webhook server.
+You can follow [the cert manager documentation](https://cert-manager.io/docs/installation/) to install it.
```

Comment:
@yashsingh74, see that the envtest has options for you set up the configs: https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/envtest#WebhookInstallOptions.

So, I believe that we need to use the options to set it up. Also, would be nice if we could add some examples here and to link the go dev docs for further info. WDYT?

Reply:
@camilamacedo86 Please check, I hope you are referring to this change ?